### DISCUSSION TITLE
With 3.2.6, the included debian folder's control file changes the package to "nicotine"

### DISCUSSION THREAD

**Discussion by stevenpusser** [nicotine-plus/nicotine-plus#2235] [Q&A] [answered]
Instead of the previous "nicotine-plus".  If someone tries to install a nicotine-3.2.6 deb package over nicotine-plus-3.2.5, there will be an overwrite issue in dpkg, and the install halts.  

If the name change is intentional, the new control file need this field for the package:

`Breaks: nicotine-plus`

**Comment by mathiascode** [comment]
We've never used the `nicotine-plus` package name on Debian. Were you using the package from the MX Linux repos?

**Comment by stevenpusser** [comment, marked as answer]
Debian stopped packaging it after Debian 10.  I see it was nicotine there.

I actually built the last 3.2.5 package for MX Linux, taking over for a
packager that had left. I just reused their debian folder for a pre-3.2.5
git version, which had the package as nicotine-plus.  OK, I'll just revert
it to nicotine for now and in the future, and have it automatically remove
nicotine-plus with the Breaks: nicotine-plus field.

Thanks for the info.

**Reply by mathiascode** [reply]
The Debian package name is a bit inconsistent compared to other distributions, for historical reasons. Nicotine+ was forked from Nicotine in the early 2000s, and I assume nobody bothered changing the package name in Debian when they switched to the fork.

**Comment by slook** [comment]
@stevenpusser Please can you arrange to delete the old nicotine-plus 3.2.2 package from the MX `test` repo, or change it into a dummy transitional package:

https://mxrepo.com/mx/testrepo/pool/test/n/nicotine-plus/

It is no longer wanted since we are settled on using `nicotine` as you have done for 3.2.6 onwards (to match historical Debian as discussed above). 

I don't feel it matters too much if it's deletion breaks user's existing installations, as nothing depends on it and is not widely discovered yet.

It might be required for you to change nicotine-plus into a dummy transitional package first before it is possible to merge the `nicotine` package into MX `main`, but nothing depends on it so this might be unwarranted effort.

The current stable version is 3.2.8 and 3.2.9 will be released soon.

Thank you kindly for your contributions, it is very much appreciated.
